src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.Map;
+
+public class LottoResult {
+
+    private final EnumMap<Rank, Integer> lottoResult;
+
+    public LottoResult(Map<Rank, Integer> lottoResult) {
+        this.lottoResult = new EnumMap<>(lottoResult);
+    }
+
+    public int calculateTotalPrize() {
+        return Rank.calculateTotalPrize(lottoResult);
+    }
+
+    public EnumMap<Rank, Integer> getResult() {
+        return lottoResult;
+    }
```

Comment:
```suggestion

    private final Map<Rank, Integer> lottoResult;

    public LottoResult(Map<Rank, Integer> lottoResult) {
        this.lottoResult = new EnumMap<>(lottoResult);
    }

    public int calculateTotalPrize() {
        return Rank.calculateTotalPrize(lottoResult);
    }

    public Map<Rank, Integer> getResult() {
        return lottoResult;
    }
```

특별한 이유가 없다면 인터페이스를 활용하는 것이 좋습니다. :)